src/RPC.elm

```diff
@@ -34,7 +34,8 @@ backendModelEndpoint _ model request =
     case Json.Decode.decodeValue Json.Decode.string request of
         Ok ok ->
             if ok == Env.adminPassword then
-                ( Ok (Codec.encodeToValue Types.backendModelCodec model), model, Cmd.none )
+                ( Http.BadBody "todo implement backendModelEndpoint" |> Err, model, Cmd.none )
```

Comment:
As long as we are happy with this state of play... deferring to @supermario

Reply:
Yep, for later if we want it for admin.